src/SignupWithMobile.js

```diff
@@ -0,0 +1,319 @@
+// @flow
+import * as React from "react";
+import {
+  View,
+  Platform,
+  TouchableOpacity,
+  StyleSheet,
+  Dimensions,
+  ActivityIndicator,
+} from "react-native";
+import type { Props as ViewProps } from "react-native/Libraries/Components/View/View";
+import StyleSheetPropType from "react-native/Libraries/StyleSheet/StyleSheetPropType";
+import ViewStylePropTypes from "react-native/Libraries/Components/View/ViewStylePropTypes";
+import TextStylePropTypes from "react-native/Libraries/Text/TextStylePropTypes";
+import type { LayoutEvent } from "react-native/Libraries/Types/CoreEventTypes";
+import {
+  parseIncompletePhoneNumber,
+  parsePhoneNumber,
+} from "libphonenumber-js";
+
+import Text from "./Text";
+import type { Props as TextProps } from "./Text";
+import TextInput from "./TextInput";
+import type { Props as TextInputProps } from "./TextInput";
+import CountryPicker from "./CountryPicker";
+import type { Props as CountryPickerProps } from "./CountryPicker";
+import { ViewStyle, TextStyle } from "./styles";
+
+const defaultStyles = StyleSheet.create({
+  container: {
+    flex: 1,
```

Comment:
I dont think giving flex 1 here is a good choice. This component should not have constraint on width and height because caller should be able to config this component's size.

Reply:
The problem is the design required the box need to be middle of the screen,
If we want to let caller control the size and position of this component.
I think I should remove the header of this component, just leave the box part.
I think it will make the caller more easy to control the layout, and we don't need to adjust the layout.